docs/machine-learning/automate-training-with-model-builder.md

```diff
@@ -68,17 +59,21 @@ Regression is used to predict numbers.
 
 ![Diagram showing regression examples such as price prediction, sales forecasting, and predictive maintenance](media/regression-examples.png)
 
-Price prediction can be used to predict house prices using location, size, and other characteristics of the house. It is an example of a regression model type.
+Price prediction can be used to predict house prices using location, size, and other characteristics of the house. It is an example of the regression machine learning task.
 
 You can use the price prediction template for your scenario if you want to predict a numerical value with your own dataset.
 
-#### Custom scenario (choose your model type)
+#### Classify images into categories
 
-The custom scenario allows you to manually choose your model type.
+This scenario is a special case of multiclass classification, where the input data to be categorized is a set of images.
+
+#### Custom scenario
+
+The custom scenario allows you to manually choose your scenario.
```

Comment:
Could we add an example of image classification here? Like with issue classification, price prediction, etc.